src/qli/java/com/lambdaherding/edi/qli/ch05/CountryFunctions.java

```diff
@@ -93,7 +102,9 @@ public static void writeToPropertiesSorted( Stream<Country> countries, Propertie
 	 * @param countries a list of countries
 	 * @return a fully-populated {@link WorldMap} object
 	 */
-	public static WorldMap buildWorldMap( Stream<Country> countries ) {
-		return null; // TODO
+	public static WorldMap buildWorldMap(Stream<Country> countries) {
+		WorldMap map = new WorldMap();
+		countries.forEach(c -> map.add(c));
```

Comment:
Dang, this is now making me feel foolish, after I used a custom `Collector` to build my world map, and I could have just done this.